core/variant/array.cpp

```diff
@@ -369,6 +369,39 @@ int Array::find(const Variant &p_value, int p_from) const {
 	return ret;
 }
 
+int Array::find_custom(const Callable &p_callable, int p_from) const {
+	if (_p->array.size() == 0) {
+		return -1;
+	}
+
+	int ret = -1;
+
+	if (p_from < 0 || size() == 0) {
```

Comment:
I know this is copied from `find()`, but the `size()` check is the same as the condition at the top. You could merge these 2 conditions.